package.json

```diff
@@ -4,7 +4,8 @@
   "description": "Call tracking using Twilio, TwiML and Express.js",
   "main": "index.js",
   "scripts": {
-    "test": "export NODE_ENV=test && mocha test"
+    "start":"node index",
+    "test": "NODE_ENV=test ./node_modules/.bin/mocha --timeout 4000 test"
```

Comment:
Without adding the env vars `TWILIO_ACCOUNT_SID` and so on the test suite will fail **locally**.

You right about the default timeout, it is 2000ms, but IMHO it should be enough to run the test suite without throwing any exceptions.